File: API/DTOs/Chamado/ChamadoListaDto.cs
namespace ControleChamados.DTOs
{

    public class ChamadoListaDto
    {
        public int Id { get; set; }

        public string Titulo { get; set; }

        public string Status { get; set; }

        public DateTime DataCriacao { get; set; }

        public DateTime PrazoConclusao { get; set; }

        public string? NomeServico { get; set; }

        public string? NomeSolicitante { get; set; }

        public string? NomeResponsavel { get; set; }
    }
}

File: frontend/src/DTOs/Chamado/ChamadoListaDto.ts
export default interface ChamadoListaDto {
    Id: number;
    Titulo: string;
    Status: string;
    DataCriacao: string;
    PrazoConclusao: string;

    NomeServico?: string;
    NomeSolicitante?: string;
    NomeResponsavel?: string | null;
}